### DISCUSSION TITLE
Slider get filling

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#4202] [Q&A] [answered]
Hello,

I'm using the Slider Component inside my application, but its not getting filled like the examples:

![grafik](https://user-images.githubusercontent.com/40752681/195854127-dc7f8e6d-9668-4f20-a10c-90d3974b7c92.png)

![grafik](https://user-images.githubusercontent.com/40752681/195854168-42010a59-65e2-494e-8d08-eabc82286384.png)

My code is:

`<Slider TValue="double" Min="0.0" Max="100.0" ValueChanged="OnSliderValueChanged" Value="CurrentSliderValue"></Slider>`
```

private async Task OnSliderValueChanged(Double newvalue)
    {
        if (TotalTime.HasValue)
        {
            var newPosition = newvalue * TotalTime.Value / 100.0;
            if ((paused == true) || (AudioIsPlaying == false))
            {
                await OnPlayClicked();
            }
            await _howl.Seek(soundId, newPosition);
        }
    }

protected Double CurrentSliderValue
    {
        get
        {
            Double sliderValue = 0.0;
            if ((TotalTime != null) && (CurrentPlaybackPosition != null))
            {
                sliderValue = (CurrentPlaybackPosition.Value / TotalTime.Value) * 100.0;
            }
            return sliderValue;
        }
    }
```

Any ideas, why this is not filled?

**Comment by stsrki** [comment, marked as answer]
What provider do you use? Bootstrap 4 and 5 have different styles. Check our demo for example:

- https://bootstrapdemo.blazorise.com/tests/forms
- https://bootstrap5demo.blazorise.com/tests/forms

**Reply by NeoCoderMatrix86** [reply]
Ah, I see :). I'm using the Bootstrapt 5 slider and have looked around the Boostrap 4 demo, my fault. Is there a plan to have the Bootstrap 5 slider with filling?Ah, I see :). I'm using the Bootstrapt 5 slider and have looked around the Boostrap 4 demo, my fault. Is there a plan to have the Bootstrap 5 slider with filling?

**Reply by stsrki** [reply]
We tend to stick with the default styles that come with a supported CSS framework. For any extra styles, you will need to adjust custom CSS accordingly.

**Comment by xiaozhi805157387** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
